build-utils/peggy-loader.ts

```diff
@@ -0,0 +1,15 @@
+import peggy from 'peggy';
+
+export default function peggyLoader(source) {
```

Comment:
Are there types by any chance for this?

Reply:
Good catch! Webpack does export a type for this, added.